miden/src/main.rs

```diff
@@ -1,3 +1,5 @@
+#![cfg(feature = "internal")]
```

Comment:
Does this mean that the executable file work only when `internal` feature is enabled?